README.md

```diff
@@ -76,6 +76,7 @@ Triagers ([@open-telemetry/collector-contrib-triagers](https://github.com/orgs/o
 
 - [Goutham Veeramachaneni](https://github.com/gouthamve), Grafana
 - [Ziqi Zhao](https://github.com/fatsheep9146), Alibaba
+- [Antoine Toulme](https://github.com/atoulme), Splunk
```

Comment:
Minor, but typically the list is sorted in alphabetical order by given name.

Reply:
I'll fix it when opening a PR for #17517